### PR TITLE
Open it context v4

### PR DESCRIPTION
**Pull request by JDFleury**
Added the following context values:
- givenBy
- givenTo
- opensGate

These are all relationship features. givenBy & givenTo will be for future uses linking the Authorizations to the UserProfiles & opensGate is the relationship between the Gate & the Key

### CODE REVIEW

**Review by junjou — APPROVED** [review]

File: openIt/ngsild-payloads/authorization.jsonld [hunk 1]
```diff
@@ -61,5 +41,33 @@
...
+    "object": "urn:ngsi-ld:Gate:123"
+  },
+  "givenBy": {
+    "type": "Relationship",
+    "object": "urn:ngsi-ld:UserProfile:123"
+  },
+  "givenTo": {
+    "type": "Relationship",
+    "object": "urn:ngsi-ld:UserProfile:ABC"
+  },
```
Comment (thomasBousselin) [inline openIt/ngsild-payloads/authorization.jsonld:56]: du coup sur l'app vous avez un user A qui peux autoriser un user B?

File: openIt/ngsild-payloads/authorization.jsonld [hunk 2]
```diff
@@ -61,5 +41,33 @@
         "value": "2024-08-30T11:00:00.000Z"
       }
     }
-  ]
+  ],
+  "forGate": {
+    "type": "Relationship",
+    "object": "urn:ngsi-ld:Gate:123"
+  },
```
Comment (thomasBousselin) [inline openIt/ngsild-payloads/authorization.jsonld:48]: raccourci de la relation useKey.opens (c'est redondant, mais pas de souci si ça vous simplifie la vie)

File: openIt/ngsild-payloads/authorization.jsonld (hunk 1 above)
Comment (JDFleury) [reply openIt/ngsild-payloads/authorization.jsonld:56]: Yes, exactly. User A will be giving authorization to User B to open the gate

File: openIt/ngsild-payloads/authorization.jsonld (hunk 2 above)
Comment (JDFleury) [reply openIt/ngsild-payloads/authorization.jsonld:48]: Frontend, having this link simplifies our life a lot

**Review by thomasBousselin — APPROVED** [review]

**Pull request merged**